vividus-plugin-excel/src/main/java/org/vividus/excel/ExcelSheetsExtractor.java

```diff
@@ -50,10 +51,9 @@ public ExcelSheetsExtractor(byte[] bytes) throws WorkbookParsingException
         }
     }
 
-    public ExcelSheetsExtractor(String path) throws WorkbookParsingException
+    public ExcelSheetsExtractor(InputStream inputStream) throws WorkbookParsingException
```

Comment:
either to add javadoc or accept bytes :)

Reply:
1) we have the constructor with `byte[]`
2) we don't have javadocs, since it's not considered as public API

PS: rule of thumb: close the stream if you are not sure whether the invoking method closes it

Reply:
how do we distinguish between private and public api in vividus from pov of developers?

Reply:
Public API is documented, everything else should be used at won risk